provision.sh

```diff
@@ -239,24 +256,35 @@ install_misc_tools() {
 
 install_language_servers() {
   echo ">>> Installing language servers"
-  snap install bash-language-server --classic
-  snap install typescript-language-server
+  #snap install bash-language-server --classic
+  #snap install typescript-language-server
+  apt-get install -y npm node-read-package-json
+  npm install -g typescript-language-server typescript
+  npm i -g bash-language-server
 }
 
 install_github_cli() {
   echo ">>> Installing Github CLI"
-  wget -qO- https://cli.github.com/packages/githubcli-archive-keyring.gpg >/usr/share/keyrings/githubcli-archive-keyring.gpg
-  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" >/etc/apt/sources.list.d/github-cli.list
+  curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg \
+    && sudo chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg \
+    && echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null 
   apt-get update
   apt-get install -y gh
+
+    
 }
 
 install_helm3() {
   echo ">>> Installing Helm 3"
-  wget -qO- https://baltocdn.com/helm/signing.asc | gpg --dearmor >/usr/share/keyrings/helm.gpg
-  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" >/etc/apt/sources.list.d/helm-stable-debian.list
+  curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/helm.gpg > /dev/null
+  apt-get install apt-transport-https --yes
+  echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/trusted.gpg.d/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+  chmod 644 /etc/apt/trusted.gpg.d/helm.gpg
   apt-get update
   apt-get install -y helm
+
+
+  curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
```

Comment:
Do we need this, given that we have just installed `helm` using `apt-get` above?

Reply:
no, was a leftover